iina/InfoDictionary.swift

```diff
@@ -70,13 +70,6 @@ struct InfoDictionary {
 
   let dictionary = Bundle.main.infoDictionary!
 
-  /// A Boolean value that indicates whether this executable was an optimized (not debug) build.
-  #if DEBUG
-  let isDebug = true
-  #else
-  let isDebug = false
-  #endif
-
```

Comment:
Pretty sure this was just added recently, @low-batt you should probably look at this and see if you needed it

Reply:
Yes, this was recently added as a part of issue #4383 / PR #4385. However my role in that issue was to just get it started and hand it off to @uiryuu. Will have to ask @uiryuu about this change.

Reply:
Let keep this, although there isn't any code using this. We might want to use this in the future.